ChangeLog.adoc

```diff
@@ -11,6 +11,7 @@ Laurent Petit <laurent dot petit at gmail dot com>
 == Changes between Counterclockwise 0.32.0 and 0.33.0
 
 - Command line options: `-Dccw.nrepl.port=[1-65535]` allows to select the embedded nrepl port. `ccw.autostartrepl` has been renamed to `ccw.nrepl.autostart` for coherence.
+- Command line options: `-Dccw.nrepl.cider.enable` adds full support for Cider to the embedded nRepl.
```

Comment:
The more I think about it, the more I find this is worth a Preference in the REPL Preference pages instead of just a command line option.